sys-apps/inxi/inxi-2.3.8_p20170114.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_COMMIT=af0630e3067c138893e243896e1767c93d2a2856 #because upstream refuses to tag commits with version numbers
+MY_SHORTCOMMIT=${MY_COMMIT:0:7}
+
+DESCRIPTION="Commandline script to print hardware information for irc and administration"
+
```

Comment:
Please group the variable like in the common set of ebuilds. That is, DESC, HOMEPAGE and SRC_URI together ;-).